sullivan_a03c.js

```diff
@@ -0,0 +1,16 @@
+/*  Anna Sullivan
+    sullivan_a03c.js
+    INFO2124WW
+    Thoendel
+    12/20/19
+*/
+
+{
+let preambleString = "When in the course of human events";
+
+console.log(`The preamble is:`);
+console.log(`\t ${preambleString}`);
+console.log("");
+console.log(`Does the preamble contain 'a'?`);
+console.log(`\t ${preambleString.includes("a")}`);
```

Comment:
Again - I'd like to see this be a single template literal. Being able to contain and handle long strings is kind of the point.